help.txt

```diff
@@ -1,33 +1,32 @@
 MANAGE PLUGINS
-  asdf plugin add <name> [<git-url>]       Add a plugin from the plugin repo OR, add a Git repo
-                                           as a plugin by specifying the name and repo url
-  asdf plugin list [--urls] [--refs]       List installed plugins. Optionally show git urls and git-ref.
-  asdf plugin list all                     List plugins registered on asdf-plugins repository with URLs
-  asdf plugin remove <name>                Remove plugin and package versions
-  asdf plugin update <name> [<git-ref>]    Update a plugin to latest commit or a particular git-ref.
-  asdf plugin update --all                 Update all plugins
+asdf plugin add <name> [<git-url>]      Add a plugin from the plugin repo OR, add a Git repo as a plugin by specifying the name and repo url
+asdf plugin list [--urls] [--refs]      List installed plugins. Optionally show git urls and git-ref
+asdf plugin list all                    List plugins registered on asdf-plugins repository with URLs
+asdf plugin remove <name>               Remove plugin and package versions
+asdf plugin update <name> [<git-ref>]   Update a plugin to latest commit or a particular git-ref
+asdf plugin update --all                Update all plugins
 
 
 MANAGE PACKAGES
-  asdf install [<name> <version>]          Install a specific version of a package or,
-                                           with no arguments, install all the package
-                                           versions listed in the .tool-versions file
-  asdf uninstall <name> <version>          Remove a specific version of a package
-  asdf current                             Display current version set or being used for all packages
-  asdf current <name>                      Display current version set or being used for package
-  asdf where <name> [<version>]            Display install path for an installed or current version
-  asdf which <command>                     Display the path to an executable
-  asdf shell <name> <version>              Set the package version in the current shell
-  asdf local <name> <version>              Set the package local version
-  asdf global <name> <version>             Set the package global version
-  asdf list <name>                         List installed versions of a package
-  asdf list all <name>                     List all versions of a package
+asdf install [<name> <version>]         Install a specific version of a package or, with no arguments, install all the package versions listed in the .tool-versions file
+asdf install <name> latest[:<version>]  Install the latest stable version of a package, or with optional version, install the latest stable version that begins with the given string
+asdf uninstall <name> <version>         Remove a specific version of a package
+asdf current                            Display current version set or being used for all packages
```

Comment:
This is what I see with wrapping text. I find it hard to follow when it wraps all the way around. <img width="736" alt="Screen Shot 2020-06-15 at 12 25 30 PM" src="https://user-images.githubusercontent.com/1520926/84682255-60adff80-af03-11ea-95ff-e490744c2c01.png">